incubator/hnc/Makefile

```diff
@@ -182,21 +182,17 @@ kind-deploy:
 	CONFIG=kind $(MAKE) deploy
 
 ###################### E2E TEST #########################
-# Sleep time is added for the webhook to get ready before test. The time 
-# comes from watching the log as well as multiple experiments. 
+# This test will run on the current cluster the user deployed (either kind or 
+# a kubernetes cluster). Please make sure to have the test cluster ready 
+# before running the test.
+
 .PHONY: e2e-test
-e2e-test: deploy
+e2e-test: 
+	@echo "Please make sure that the webhook server is ready."
+	@echo "It takes around 20 seconds for the webhook server to get ready in a fresh cluster."
```

Comment:
This could be confusing, because it seems to ask users to do something but usually, nothing will be required.

How about something like: "If these tests fail due to the webhook not being ready, wait 30s and try again. Note that webhooks can take up to 30s to become ready after HNC is first deployed in a cluster."

Reply:
Done